src/components/HomeComponents/Hero.jsx

```diff
@@ -57,28 +57,37 @@ const DemoControls = styled.div`
     flex: 0 0 100%;
     margin: 0;
 
+    ${Button} {
+      border-radius: 0;
+      width: 100%;
+      margin: 0;
+    }
+
     ${({ theme }) => theme.media.medium`
-      margin: 0 0.6em 0 4em;
       padding: 0;
       flex: 1;
     `}
   }
 `;
 
 const HomeHero = props => (
-  <Hero themeName={props.themeName}>
+  <Hero>
     <Container>
       <H1>Make your player yours</H1>
       <H2>with the worlds most popular open source HTML5 player</H2>
 
       <DemoContainer>
-        <Player
-          controls
-          fluid
-          themeName={props.themeName}
-          sources={[{ src: props.video, type: 'application/x-mpegurl' }]}
-          poster={props.poster}
-        />
+        {
+          props.heroTheme ?
```

Comment:
I was suggesting we not try to SSR the player, which feels like a better fix for this one. We could either make that check here (`typeof window !== undefined && /* ... */`) or not initialize Video.js in the `Player` constructor, but I think making the check here is 👍

Reply:
When I tried that it did not fix the bug. The double-render on the client side was still causing the same issue. I have a separate branch you can see

dj/redesign-theme-picker-5

Maybe I missed something? Your explanation made sense, but when I implemented that it did not fix it.